modules/imap/output_modules.php

```diff
@@ -901,6 +901,7 @@ protected function output() {
         $reset = '';
         if (array_key_exists('simple_msg_parts', $settings) && $settings['simple_msg_parts']) {
             $checked = ' checked="checked"';
+        } else {
```

Comment:
This else means that the reset checkbox below will be without label - seems like a bug to me. What is the purpose? If the checkbox shouldn't be visible at all, then you should hide the checkbox as well.

Reply:
Ok, I see!